tests/test_pyglmnet.py

```diff
@@ -415,8 +423,8 @@ def test_random_state_consistency():
     glm_b = GLM(distr="gaussian", random_state=1)
     ypred_b = glm_b.fit_predict(Xtrain, ytrain)
     match = "This glm object has already been fit"
-    with pytest.raises(ValueError, match=match):
-        ypred_c = glm_b.fit_predict(Xtrain, ytrain)
+    # with pytest.raises(ValueError, match=match):
+    #     ypred_c = glm_b.fit_predict(Xtrain, ytrain)
```

Comment:
why is this test commented out?

Reply:
We don't have the same functionality of re-fitting error now so I commented out.

Reply:
right I see, okay with removing this test since we removed the error related to re-fitting. Now since you moved the init of the attributes to fit, I think there is no danger with refitting any more. This seems like the right way of doing things.

Reply:
I guess I can mark this as resolved for now!

Reply:
almost, just delete the lines. We don't keep commented code in our repo unless it's to explain something :)